### DISCUSSION TITLE
How long will version 2.x be maintained?

### DISCUSSION THREAD

**Discussion by comcy** [elsa-workflows/elsa-core#5248] [General]
Regading the latest 3.x releases and the older version of 2.x. 
Is there any information from your side how long it is planned to maintain the version 2.x.
Just in concerns of security, bugfixing, ...

I did not find any information about on the github sites or nuget.org.

Thanks in advance!

**Comment by glime-ai** [comment]
As of my knowledge cutoff date in early 2023, there is no specific information provided regarding the end-of-life or maintenance period for Elsa 2.x. However, it is common practice for software maintainers to eventually phase out support for older versions as newer versions become stable and widely adopted.

Given that Elsa 3.0 represents a significant leap from its predecessor and is described as a complete rewrite with many improvements and new features, it is likely that the focus of the development team will shift towards the newer version. This could mean that support for Elsa 2.x may diminish over time.

For the most accurate and up-to-date information, I would recommend reaching out directly to the Elsa maintainers through their official communication channels, such as their GitHub repository or Discord server. They would be able to provide you with the latest information on their support plans for Elsa 2.x.

If you decide to stay with Elsa 2.x, it's important to monitor the official Elsa GitHub repository for any announcements regarding security patches or bug fixes. Additionally, consider planning for an eventual migration to Elsa 3.x to ensure that you continue to receive updates and support in the future.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
